docs/user/aws/install_upi.md

```diff
@@ -19,6 +20,29 @@ $ openshift-install-linux-amd64 create ignition-configs
 ? Pull Secret [? for help]
 ```
 
+Edit the resulting `openshift-install.yaml` to set `replicas` to 0 for the `compute` pool, which will remove the compute MachineSets.
+Then remove the control-plane Machines directly with:
+
+```console
+$ openshift-install create manifests
+WARNING There are no compute nodes specified. The cluster will not fully initialize without compute nodes.
+INFO Consuming "Install Config" from target directory
+```
+
+We already know we're taking responsibility for the compute nodes, so the warning is not news.
+From the manifest assets, remove the control-plane Machines:
+
+```console
+$ rm -f openshift/99_openshift-cluster-api_master-machines-*
```

Comment:
nit: can we make this highly specific like maybe include the `.yaml` ?

Reply:
> nit: can we make this highly specific like maybe include the `.yaml` ?

Done with a6cdf1abc -> 84e5a807a.